test/simulator/misc/decisions.js

```diff
@@ -393,12 +401,20 @@ describe('Choices', function () {
 			]]);
 
 			battle.makeChoices('move lunardance, move lunardance', 'move lunardance, move lunardance');
-			battle.sides.forEach(side => side.active.forEach(pokemon => assert.fainted(pokemon)));
+			for (const side of battle.sides.values()) {
```

Comment:
FYI this causes tests to fail for developers who are on Node versions older than 10.9. https://node.green/#ES2015-built-in-extensions-Array-prototype-methods-Array-prototype-values. https://github.com/Zarel/Pokemon-Showdown/pull/5282 to fix.